tests/functionaltests/FunctionalTests.cpp

```diff
@@ -28,6 +28,14 @@ MODULE_PROPERTY(L"UAP:Host", L"Xaml")
 MODULE_PROPERTY(L"UAP:AppXManifest", L"Default.AppxManifest.xml")
 END_MODULE()
 
+// This is a method that UIKit exposes for the test frameworks to use.
+extern "C" void UIApplicationInitialize(const wchar_t*, const wchar_t*);
+
+void UIApplicationDefaultInitialize() {
+    // Pass null to indicate default app and app delegate classes
+    UIApplicationInitialize(nullptr, nullptr);
+}
+
```

Comment:
yeah sorry bad merge on this file. I just wanted to delete everything in it basically but y'all made a bunch of changes to it.